src/applications/facility-locator/utils/helpers.jsx

```diff
@@ -14,6 +17,18 @@ export const setFocus = selector => {
   }
 };
 
+/**
+ * Helper fn to set click listeners for recording map zoom events +/-
+ */
+export const setZoomEvents = () => {
+  const recordZoomIn = () => recordEvent({ event: 'fl-map-zoom-in' });
+  const recordZoomOut = () => recordEvent({ event: 'fl-map-zoom-out' });
+  const zoomIn = document.querySelector('.leaflet-control-zoom-in');
+  if (zoomIn) zoomIn.addEventListener('click', recordZoomIn);
```

Comment:
Is there a chance the map hasn't finished loading yet when this is called?

Reply:
it's setting the event listener only and is being set on `componentDidMount()` ,  all should be rendered right?

Reply:
but I see the issue now, it's not triggering, will check

Reply:
`componentDidMount` doesn't guarantee the map has finished loading.